File: Descending Order/descending-order.js
/**
 * Return it with its digits in descending order
 *
 * @param {int} n
 * @return {int}
 */
const descendingOrder = n => {
  return parseInt(n.toString().split('').sort((a, b) => b - a).join(''));
};

console.log(descendingOrder(1021)); // 2110
console.log(descendingOrder(123456789)); // 987654321

File: Descending Order/descending-order.py
def Descending_Order(num):
    """
    :type num: int
    :rtype: int
    """
    return int(''.join(sorted([x for x in str(num)], key=int, reverse=True)))


print(Descending_Order(1021))  # 2110
print(Descending_Order(123456789))  # 987654321